plugins/modules/network_settings_intent.py

```diff
@@ -1898,10 +1913,10 @@ def delete_reserve_pool(self, name):
             self.status = "success"
             return self
 
-        self.log("Reserved Ip Pool to be deleted " +
-                 str(self.have.get("reservePool").get("name")))
+        self.log("Reserved Ip Pool to be Deleted: {0}"
```

Comment:
self.log("Reserved IP Pool scheduled for deletion: {0}"
          .format(self.have.get("reservePool").get("name")), "INFO")